e2e/tests/api-driven/src/flowStatusHistory/helpers.ts

```diff
@@ -0,0 +1,72 @@
+import { FlowStatus } from "@opensystemslab/planx-core/types";
+import { $admin } from "../client";
+import { createTeam } from "../globalHelpers";
+import gql from "graphql-tag";
+
+export const setup = async () => {
+  const teamId = await createTeam();
+
+  const world = { teamId };
+
+  return world;
+};
+
+export const cleanup = async () => {
+  await $admin.user._destroyAll();
+  await $admin.flow._destroyAll();
+  await $admin.team._destroyAll();
+};
+
+interface GetFlowStatus {
+  flows: {
+    status: FlowStatus;
+  }[];
+}
+
+export const getFlowStatus = async (flowId: string) => {
+  const {
+    flows: [{ status }],
+  } = await $admin.client.request<GetFlowStatus>(
+    gql`
+      query GetFlowStatus($flowId: uuid!) {
+        flows(where: { id: { _eq: $flowId } }) {
```

Comment:
nit: is `flows_by_pk` slightly more performant & descriptive here?

Reply:
Good catch thanks - fixed in [7c43879](https://github.com/theopensystemslab/planx-new/pull/3170/commits/7c43879c50d3639f7a2eb0e70764eba128c9fd4a)